Misc/NEWS.d/next/Library/2024-02-11-20-23-36.gh-issue-114563.RzxNYT.rst

```diff
@@ -0,0 +1,4 @@
+Fix several `format()` bugs when using the C implementation of `Decimal`:
+* memory leak in some rare cases when using the `z` format option (coerce negative 0)

```

Comment:
It is reStructuredText, not Markdown. Use either double backquotes ``` ``z`` ```, or simple quotes 'z', or no quotes at all.